From 0ab1d5a6a04a6ba7aec1cfd726177c395cf11167 Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@gmail.com>
Date: Thu, 4 Sep 2025 15:32:09 -0400
Subject: [PATCH] Update kvim4_defconfig

Additions:
CONFIG_SD_BOOT
CONFIG_CMD_SETEXPR
CONFIG_CMD_UUID
CONFIG_CMD_FS_UUID

Signed-off-by: Patrick Yavitz <pyavitz@gmail.com>
---
 board/khadas/defconfigs/kvim4_defconfig | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/board/khadas/defconfigs/kvim4_defconfig b/board/khadas/defconfigs/kvim4_defconfig
index 4af2b420..515d1471 100644
--- a/board/khadas/defconfigs/kvim4_defconfig
+++ b/board/khadas/defconfigs/kvim4_defconfig
@@ -14,6 +14,7 @@ CONFIG_ARMV8_CRYPTO=y
 CONFIG_DEBUG_UART=y
 # CONFIG_ANDROID_BOOT_IMAGE is not set
 CONFIG_OF_BOARD_SETUP=y
+CONFIG_SD_BOOT=y
 CONFIG_BOOTDELAY=2
 CONFIG_BOARD_LATE_INIT=y
 # CONFIG_DISPLAY_CPUINFO is not set
@@ -27,6 +28,7 @@ CONFIG_AUTOBOOT_STOP_STR=" "
 # CONFIG_CMD_DSPSET is not set
 CONFIG_CMD_GO=y
 # CONFIG_CMD_IMI is not set
+# CONFIG_CMD_SAVEENV is not set
 CONFIG_CMD_CLKMSR=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
@@ -39,16 +41,16 @@ CONFIG_CMD_SF=y
 CONFIG_CMD_SF_TEST=y
 CONFIG_CMD_USB=y
 CONFIG_CMD_SARADC=y
-# CONFIG_CMD_SETEXPR is not set
 CONFIG_CMD_BMP=y
+CONFIG_CMD_UUID=y
 CONFIG_CMD_BTRFS=y
+CONFIG_CMD_FS_UUID=y
 CONFIG_BOOTLOADER_CONTROL_BLOCK=y
 CONFIG_CMD_BOOTCTOL_AVB=y
 CONFIG_CMD_WATCHDOG=y
 CONFIG_CMD_JTAG=y
 CONFIG_CMD_INI=y
 CONFIG_CMD_CFGLOAD=y
-# CONFIG_CMD_SAVEENV is not set
 CONFIG_AML_PARTITION=y
 CONFIG_OF_CONTROL=y
 CONFIG_OF_EMBED=y
-- 
2.43.0

